src/site/layouts/media_list_images.drupal.liquid

```diff
@@ -8,88 +8,94 @@
   <main class="va-l-detail-page">
     <div class="usa-grid usa-grid-full">
       <div class="usa-width-three-fourths">
+        <!-- Draft status -->
         {% if !entityPublished %}
-          <div class="usa-alert usa-alert-info">
-            <div class="usa-alert-body">
-              <p class="usa-alert-text">You are viewing a draft.</p>
-            </div>
+        <div class="usa-alert usa-alert-info">
+          <div class="usa-alert-body">
+            <p class="usa-alert-text">You are viewing a draft.</p>
           </div>
+        </div>
         {% endif %}
 
         <article class="usa-content">
           <!-- Title -->
           <h1>{{ title }}</h1>
 
           <!-- Intro -->
-          <p>{{ fieldIntroTextLimitedHtml.processed }}</p>
+          <div class="va-introtext">{{ fieldIntroTextLimitedHtml.processed }}</div>
 
           <!-- Alert -->
           {% if fieldAlertSingle %}
-            {% include "src/site/paragraphs/alert_single.drupal.liquid" with entity = fieldAlertSingle.entity %}
+          {% include "src/site/paragraphs/alert_single.drupal.liquid" with entity = fieldAlertSingle.entity %}
           {% endif %}
 
           <!-- Buttons -->
-          {% for fieldButton in fieldButtons %}
+          <div class="vads-u-margin-y--3">
+            {% for fieldButton in fieldButtons %}
             {% include "src/site/paragraphs/button.drupal.liquid" with entity = fieldButton.entity %}
```

Comment:
can we indent the include here?

Reply:
So this is actually how our linter is set up for liquid templates, it always remove the indentation 🤔 not sure how we can modify that as I haven't dealt too much with liquid